tests/test_cli.py

```diff
@@ -293,6 +293,43 @@ def test_send_transaction_to_contract_from_compiled_contracts(
     assert is_0x_prefixed(transaction_hash)
 
 
+@pytest.mark.usefixtures("go_to_root_dir")
+def test_send_transaction_with_value_parameter(
+    runner, test_contract_address, test_contract_name
+):
+    """Change balance of a contract.
+
+    Test the transaction value parameter by a payable function on the
```

Comment:
this should be a comment, not a docstring.

Reply:
:heavy_check_mark: